### DISCUSSION TITLE
RepOOS in 2.0?

### DISCUSSION THREAD

**Discussion by gabefishbein** [amzn/selling-partner-api-models#3324] [General] [answered]
Anyone seen the RepOOS metric in 2.0? Noticed it's missing and was hoping if anyone had any idea if it was going to be added. We use this to determine if there are supply chain issues.

**Comment by sungolivia** [comment, marked as answer]
Hey @gabefishbein there is a plan to add ROOS(Retail/Procurable Out of Stock) but no confirmed date yet. Check back around July and I will provide an update if there is anything as well!
